packages/backend/services/dbFirebase.js

```diff
@@ -38,9 +39,33 @@ const findUserByAddress = async builderAddress => {
   return { exists: true, data: { id: builderSnapshot.id, ...builderSnapshot.data() } };
 };
 
+// --- Events
+const createEvent = event => {
+  return database.collection("events").add(event);
+};
+
+const findAllEvents = async ({ limit: limitArg } = {}) => {
+  let eventsSnapshot;
+  if (limitArg) {
+    eventsSnapshot = await database.collection("events").limit(limitArg).get();
+  } else {
+    eventsSnapshot = await database.collection("events").get();
+  }
+
+  return eventsSnapshot.docs.map(doc => doc.data());
+};
```

Comment:
would be nice to be able to send `.limit(undefined)` but it looks like firebase doesn't allow that :(